models/issues/issue_project.go

```diff
@@ -13,28 +13,22 @@ import (
 )
 
 // LoadProject load the project the issue was assigned to
-func (issue *Issue) LoadProject(ctx context.Context) (err error) {
-	if issue.Project == nil {
-		var p project_model.Project
-		has, err := db.GetEngine(ctx).Table("project").
+func (issue *Issue) LoadProjects(ctx context.Context) (err error) {
+	if len(issue.Projects) == 0 {
```

Comment:
It causes duplicate "loads".

If should check `issue.Projects == nil`, and below it should set `issue.Projects` to an empty slice but not nil if there is no project.